### PR TITLE
feat(email-templates) - revert inline image

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
What's happening here?

**Review by searchableguy — APPROVED** [review]

File: packages/email-templates/src/templates/keyAirdropped.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ export default {
 
 {{#if customContent}}
   <section style="margin: 18px 0px; padding: 18px 18px 10px 18px; background: #F8FAFC; font-family: monospace, sans-serif;" class="custom">
-    {{#if lockImage}}<img src="{{inlineImage lockImage}}" style="max-height: 40px; float: left; margin-right: 5px;" >{{/if}}
+    {{#if lockImage}}<img src="{{ lockImage}}" style="max-height: 40px; float: left; margin-right: 5px;" >{{/if}}
```
Comment (julien51) [inline packages/email-templates/src/templates/keyAirdropped.ts:14]:
```suggestion
```

File: packages/email-templates/src/templates/keyMined.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@ export default {
 
 {{#if customContent}}
   <section style="margin: 18px 0px; padding: 18px 18px 10px 18px; background: #F8FAFC; font-family: monospace, sans-serif;" class="custom">
-    {{#if lockImage}}<img src="{{inlineImage lockImage}}" style="max-height: 40px; float: left; margin-right: 5px;" >{{/if}}
+    {{#if lockImage}}<img src="{{ lockImage }}" style="max-height: 40px; float: left; margin-right: 5px;" >{{/if}}
```
Comment (julien51) [inline packages/email-templates/src/templates/keyMined.ts:14]:
```suggestion
```

**Review by julien51 — COMMENTED** [review]
Please remove lock images completely

**Review by julien51 — APPROVED** [review]

**Pull request merged**